runtime/kusama/src/lib.rs

```diff
@@ -1003,86 +1002,95 @@ impl Default for ProxyType {
 	}
 }
 
-impl InstanceFilter<Call> for ProxyType {
-	fn filter(&self, c: &Call) -> bool {
+impl InstanceFilter<RuntimeCall> for ProxyType {
```

Comment:
lines to double check by other reviewers.

Reply:
what needs to be double checked?